02-Reproject-Raster-In-R.Rmd

```diff
@@ -128,16 +140,15 @@ plot(DTM_HARV,
 
 ```
 
-Our results are curious - the Digital Terrain Model (`DTM_HARV~) did not plot on 
+Our results are curious - the Digital Terrain Model (`DTM_HARV`) did not plot on
 top of our hillshade. The hillshade plotted just fine on it's own. Let's try to 
 plot the DTM on it's own to make sure there are data there.
 
 ```{r plot-DTM }
-
 #Plot DTM 
 plot(DTM_HARV,
      col=terrain.colors(10),
-     alpha=0.4,
+     alpha=1,
```

Comment:
alpha =1 is opaque not transparent.  Changed 0.4 to 1 to increase color as it was being plotted alone to make sure the layer worked (part of re-project lesson).